src/test/java/seedu/address/model/schedule/ScheduleTest.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.model.schedule;
+
+public class ScheduleTest {
```

Comment:
Thanks for creating the Schedule Test file - We will work on this after we finalized the features